backend/initial_setup.py

```diff
@@ -198,3 +206,20 @@ def get_version(cls):
             ]).decode().strip()
         except subprocess.CalledProcessError:
             return 'unknown'
+
+
+@authorizer.verify_password
+def verify_password(username, password):
+    if username in USERS:
+        return USERS[username] == password
+    return False
+
+
+def login_decorator(decorated_function):
```

Comment:
Why not put this in the decorators.

Reply:
Because it uses `USERS` and is only used in one place (this class)